### PR TITLE
Functional tests - Refacto the BO - Invoices -  generate invoice by status test

### PR DESCRIPTION
**Pull request by khouloudbelguith**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Refacto the BO - Invoices -  generate invoice by status test
| Type?             | refacto
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | HEADLESS=false TEST_PATH="functional/BO/02*/02*/02*" URL_FO="http://shop.com/" PASSWD="prestashop_demo" npm run test:specific
| Possible impacts? | no

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/02_orders/02_invoices/02_generateInvoiceByStatus.js [hunk 1]
```diff
@@ -131,5 +131,19 @@ describe('BO - Orders - Invoices : Generate PDF file by status', async () => {
       const exist = await files.doesFileExist(filePath);
       await expect(exist).to.be.true;
     });
+
+    it('should choose one status, generate the invoice and check the file existence', async function () {
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/02_invoices/02_generateInvoiceByStatus.js:135]: @khouloudbelguith Is it normal the duplication with the previous test ?

**Review by Progi1984 — COMMENTED** [review]
A small question

File: tests/UI/campaigns/functional/BO/02_orders/02_invoices/02_generateInvoiceByStatus.js (hunk 1 above)
Comment (khouloudbelguith) [reply tests/UI/campaigns/functional/BO/02_orders/02_invoices/02_generateInvoiceByStatus.js:135]: @Progi1984, I'm updating the automatic tests according to the scenario written and approved in the Jira.
The first test is selecting two statuses and the second is about selecting only one status.
Thanks!

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by matks** [comment]
Thank you @khouloudbelguith

**Pull request merged**